cluster/aws/vivid/util.sh

```diff
@@ -34,6 +34,10 @@ function detect-vivid-image () {
         AWS_IMAGE=ami-ee023e80
         ;;
 
+      ap-northeast-2)
+        AWS_IMAGE=ami-c2df11ac
```

Comment:
@justinsb great.
when I looked before I couldn't see `ap-northeast-2` on the image finder,
seems I overlooked it right at the bottom of the select box.